File: C++Programming/Practice/program7.cpp
// Program to check whether number is even or odd

#include<iostream>
using namespace std;

bool CheckEvenOdd(int iNo)
{
    int iRemender = 0;

    iRemender = iNo % 2;

    if(iRemender == 0)
    {
        return true;
    }
    else
    {
        return false;
    }

}
int main()
{
    int iValue = 0;
    bool bRet = false;
    
    cout<<"Enter a number : ";
    cin>>iValue;

    bRet = CheckEvenOdd(iValue);

    if(bRet == true)
    {
        cout<<iValue<<" is even\n";
    }
    else
    {
        cout<<iValue<<" is Odd\n";
    }

    return 0;
}

File: Java_Programming/Practice/program7.java
// Check whether number is even or odd

import java.util.Scanner;

class program7
{
    public static void CheckEvenOdd(int iNo)
    {
        int iRemainder = 0;

        iRemainder = iNo % 2;

        if(iRemainder == 0)
        {
            System.out.println("Number is Even");
        }

        else
        {
            System.out.println("Number is Odd");
        }

    }

    public static void main(String A[])
    {
        int iValue = 0;
        
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter number : ");
        iValue = sc.nextInt();

        CheckEvenOdd(iValue);

    }

}